flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+  "only-arches": ["x86_64"]
```

Comment:
this is disappointing

Reply:
yes, but https://build.opensuse.org/package/live_build_log/games/urbanterror/openSUSE_Tumbleweed/aarch64
> code/server/../qcommon/q_platform.h:272:2: error: #error "Architecture not supported"